aiida/orm/implementation/general/calculation/job/__init__.py

```diff
@@ -50,7 +50,6 @@ class JobCalculationFinishStatus(enum.Enum):
     RETRIEVALFAILED = 200
     PARSINGFAILED = 300
     FAILED = 400
-    I_AM_A_TEAPOT = 418
```

Comment:
Interesting old state ;-)

Reply:
Yeah, I put that in when it was still the `workflows` branch. Thought it best to just remove it now before someone gets hurt :)